README.md

```diff
@@ -41,6 +41,16 @@ Do `make build` every time you would like to pick up and install new local/proje
 
 The first time it will also set up your pre-commit hooks.
 
+### Proxygen - WIP
+
+if you wish to deploy proxies to Apigee then there is some additional setup outside of the above due to proxygen not being an ASDF plugin
+
+`pip install proxygen-cli`
```

Comment:
add this to `pyproject.toml` with `poetry proxygen-cli --group dev`

Reply:
Cant do this unfortunately, its not a recognised package for poetry - only way ive managed to install it is with the above